pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -268,6 +269,14 @@ async def run(self):
             except CalledProcessError as ex:
                 _LOGGER.warning(f"Unable to move {impetus} advisory {advisory} to QE: {ex}")
 
+        _LOGGER.info("Check failed greenwave tests ...")
+        self._slack_client.bind_channel(self.release_name)
```

Comment:
Would this be posted in a separate message in the release channel or as a single message in the prepare-release thread? Would prefer the second option

Reply:
not in thread

Reply:
the slack thread is created in the Jenkinsfile so there's no good way to access it in python. i think pulling that into python would be a good separate change. then this could be a thread response.